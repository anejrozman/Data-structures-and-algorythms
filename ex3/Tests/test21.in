485,482,126
5,7,1,5,8,2,8,5,2,4,3,10,5,2,4,3,8,2,5,4,9,8,3,8,7,6,2,7,8,4,5,7,2,8,4,6,3,2,1,3,7,8,10,3,10,3,4,5,3,3,8,4,9,6,2,6,6,2,10,5,2,5,2,4,3,4,1,6,7,3,1,6,10,10,3,9,10,10,10,2,1,10,1,8,10,7,3,4,2,9,5,7,4,5,7,4,8,9,6,2,8,3,9,6,2,8,1,8,3,10,2,7,5,6,1,2,3,7,4,4,10,4,4,3,5,8,3,4,10,9,4,1,1,8,7,2,3,10,5,10,1,6,5,1,9,5,4,6,7,4,1,3,6,1,9,9,9,2,7,5,10,3,5,3,1,2,7,7,8,6,6,3,9,1,6,6,3,10,1,5,6,5,3,7,10,5,6,7,4,8,8,4,8,8,8,7,6,9,7,4,8,4,7,2,2,4,1,4,2,3,4,1,9,4,10,6,4,9,5,7,6,9,10,1,3,3,9,9,1,10,2,9,3,5,3,6,8,1,1,4,9,5,3,7,5,7,8,1,5,9,3,1,8,9,7,2,10,2,7,6,1,2,7,1,10,2,9,7,6,4,4,3,9,5,9,6,1,1,9,10,4,1,9,10,6,9,9,7,6,7,10,4,8,8,5,10,4,8,2,9,2,8,7,9,8,10,5,4,2,8,7,2,8,8,8,7,8,4,8,9,8,6,9,5,2,4,4,1,9,4,2,8,4,8,9,9,4,2,6,7,2,7,1,4,6,2,4,3,5,5,4,8,4,2,8,3,5,7,9,5,3,7,10,4,7,1,1,4,2,8,8,9,7,6,7,9,1,4,9,10,7,2,2,2,2,3,2,3,6,5,3,2,7,10,3,1,8,4,10,4,6,7,4,4,2,1,7,9,2,7,7,6,10,6,1,9,7,1,7,7,3,5,2,1,10,8,9,2,10,6,10,6,5,5,7,10,5,6,4,2,8,6,8,1,3,9,7,9,10,2,8,5,3,2,2,2,10,10,1,9,4,1,9,4,3,6,5,10,3,7,4,9,7,1,10,2,9,2,9,8,2,7,5,1,9
429,193,3.115
31,373,3.423
42,248,5.935
120,376,9.726
284,67,11.805
405,458,12.931
100,204,15.08
383,298,15.706
147,120,19.477
91,306,22.312
204,96,27.213
374,166,29.647
229,218,35.825
182,202,40.233
160,260,41.86
329,204,42.954
15,181,42.526
267,150,48.115
411,421,50.028
88,295,55.992
347,390,58.917
252,424,59.386
70,301,60.373
376,237,60.148
24,363,62.517
476,400,63.015
43,383,68.373
340,236,68.159
141,323,71.699
455,362,71.509
458,321,71.12
102,295,75.415
290,402,82.294
352,375,85.669
224,47,86.452
335,336,87.527
137,171,90.818
29,387,91.309
99,453,93.075
82,468,94.966
40,88,101.168
320,381,101.043
480,313,101.42
301,372,105.369
114,368,106.055
382,243,107.013
269,249,105.583
9,286,105.569
460,58,110.388
212,156,110.186
336,477,105.714
180,150,113.105
31,13,119.984
299,319,120.681
0,242,119.012
408,324,125.674
102,47,129.74
295,156,131.773
9,455,132.063
104,68,143.962
81,134,144.752
104,84,146.736
99,460,147.86
393,260,146.962
58,197,149.648
309,325,150.328
126,244,151.409
285,75,159.602
246,346,159.857
126,295,161.136
316,283,162.586
291,222,161.43
5,3,164.408
229,387,165.663
2,10,168.75
61,288,174.746
157,143,177.782
210,221,179.113
285,357,180.246
122,36,182.213
131,197,185.542
422,431,189.774
377,85,190.949
97,315,191.107
441,90,192.982
75,29,194.179
112,208,196.001
9,196,198.948
405,221,199.874
83,327,200.145
266,61,202.144
358,403,203.955
234,107,208.862
331,455,209.506
102,235,208.519
123,33,214.952
141,29,214.971
448,57,216.287
91,107,216.783
408,103,218.701
459,315,219.859
275,178,214.812
320,368,214.502
234,48,227.424
46,409,229.966
124,369,230.261
251,196,232.395
146,408,233.695
196,353,232.285
346,305,232.642
396,24,236.424
156,148,237.352
294,181,236.255
277,352,239.185
397,402,240.244
3,148,240.56
453,256,239.436
434,314,247.763
254,312,248.289
157,348,251.518
30,184,253.43
410,94,255.286
9,215,256.219
215,264,257.864
437,174,258.924
412,271,259.041
382,257,255.048
145,383,262.352
168,290,263.889
135,289,264.052
278,374,263.444
254,169,265.398
127,234,269.149
114,265,270.542
138,225,274.62
430,275,275.537
305,318,279.033
308,383,282.821
402,172,286.952
146,199,290.383
42,250,292.726
109,317,293.619
338,431,295.547
178,310,296.025
81,181,306.607
51,15,311.801
373,370,311.26
426,85,313.592
465,180,311.634
228,133,315.66
417,369,318.228
8,202,318.015
277,32,321.285
251,210,321.441
308,362,323.076
414,292,323.867
139,132,325.496
82,263,326.665
232,475,332.694
106,78,335.536
170,436,336.575
77,202,336.324
441,30,336.805
355,249,341.311
404,200,352.796
85,60,353.023
249,457,353.558
484,258,354.916
140,372,353.047
113,306,356.911
79,110,358.888
159,131,359.703
136,1,361.858
359,462,362.345
244,408,368.488
366,74,373.166
145,300,376.242
117,86,380.326
85,356,386.355
166,53,387.125
326,72,386.206
10,180,386.031
204,70,395.467
461,4,400.371
426,476,401.71
25,32,401.973
255,360,403.885
474,168,404.583
258,474,404.127
468,212,415.62
288,119,416.273
194,339,417.774
170,447,418.108
117,96,419.907
207,311,421.556
61,372,421.624
310,22,426.276
332,148,431.56
446,409,432.797
275,314,434.411
478,204,436.156
375,45,437.736
308,19,438.715
178,316,439.289
347,278,440.043
83,60,441.596
132,235,442.893
308,400,443.689
314,117,445.689
224,370,446.412
149,396,447.653
363,479,448.164
27,130,450.234
34,87,454.368
131,386,456.984
117,296,457.023
414,28,458.394
5,357,456.502
70,355,456.639
395,289,462.73
131,400,464.031
291,483,470.211
125,205,471.976
453,414,478.198
228,16,479.308
78,56,478.101
213,129,482.305
331,142,483.981
41,408,485.055
106,1,487.404
16,277,487.461
37,205,489.502
312,446,490.897
417,450,488.542
375,445,491.373
318,339,491.189
342,243,487.746
319,138,495.599
475,207,494.071
428,28,497.664
21,159,497.082
260,397,503.579
390,164,506.393
209,62,507.022
137,5,507.976
63,365,509.75
77,347,514.587
264,296,516.98
465,378,519.203
26,315,521.277
16,58,529.071
394,60,530.487
463,469,530.426
117,282,530.377
464,26,529.249
115,324,534.236
441,356,536.939
48,235,537.204
202,198,540.284
41,250,541.482
353,198,540.591
368,183,543.661
307,192,545.102
253,72,546.688
420,339,545.463
157,127,549.951
87,327,552.992
26,431,553.238
352,288,555.733
253,131,558.271
113,138,559.323
383,324,562.908
110,249,564.752
179,384,564.406
398,22,569.461
321,422,570.071
66,312,581.751
354,232,585.457
375,175,588.602
461,394,592.213
462,475,594.959
160,28,596.987
272,193,597.336
41,473,596.525
13,242,598.56
253,143,600.815
130,232,600.674
3,111,603.424
287,330,605.803
345,313,607.853
137,79,610.736
282,69,614.221
389,118,615.724
40,467,616.821
478,408,617.048
193,409,614.579
290,58,621.537
274,114,627.714
371,102,627.757
362,251,627.374
282,278,632.328
417,223,633.883
359,89,636.554
391,225,638.5
204,157,639.717
273,197,641.264
190,300,643.821
96,171,646.446
198,394,646.082
46,11,647.364
213,394,651.904
24,268,660.907
30,238,660.176
234,345,662.379
221,91,664.065
122,226,664.113
198,448,667.961
164,372,673.595
408,109,675.882
274,182,676.39
226,7,680.55
413,107,682.315
472,93,683.59
299,19,683.98
353,394,683.663
439,348,686.942
320,124,690.611
185,440,691.517
332,156,693.346
410,419,698.748
349,326,702.421
207,476,703.011
271,93,707.926
116,187,708.153
456,355,708.847
477,409,715.46
90,135,716.023
252,275,717.809
337,169,719.497
422,312,720.261
421,463,721.961
367,355,723.847
155,365,724.078
320,200,725.677
102,416,728.081
431,278,731.793
309,242,731.4
471,112,737.407
33,403,739.65
114,229,745.764
73,41,747.807
292,162,748.324
311,58,748.645
65,374,754.847
15,425,761.452
392,470,761.692
227,423,764.6
98,8,769.263
158,409,769.266
2,0,769.134
239,52,770.255
85,375,776.153
125,415,777.085
300,381,776.971
253,325,779.273
210,389,782.144
179,181,783.046
402,333,785.24
427,382,786.186
88,358,787.979
180,265,789.073
417,237,790.258
122,431,790.643
383,375,794.531
434,182,794.481
126,198,797.595
347,327,799.346
104,450,800.212
155,187,804.276
344,96,805.644
87,285,806.316
108,27,807.083
316,19,807.043
46,234,810.26
369,456,812.452
357,61,814.884
126,392,815.388
242,228,815.023
6,416,817.287
254,114,818.805
158,350,819.676
377,471,820.084
278,192,821.097
1,284,827.442
118,202,828.871
419,277,830.764
379,95,831.18
289,330,830.7
4,190,833.329
149,307,834.678
279,123,835.614
46,14,836.734
183,153,836.97
104,233,834.644
167,97,842.123
479,155,843.473
366,3,846.178
337,52,848.338
356,2,853.033
41,185,860.554
356,410,861.884
169,160,861.062
23,252,865.84
401,270,874.172
0,214,875.29
152,460,874.243
146,275,875.32
441,109,879.525
350,469,880.694
333,81,881.913
89,400,880.04
50,461,883.397
476,216,884.383
281,328,885.361
401,194,886.125
133,201,884.373
129,105,892.953
141,484,893.461
82,213,898.863
81,161,899.319
185,110,900.914
264,442,901.332
439,211,898.36
403,119,903.706
90,12,903.851
419,217,905.164
150,403,902.865
72,188,910.008
370,164,911.948
101,29,917.729
343,296,918.559
114,146,918.711
224,66,920.608
61,163,928.266
207,121,931.698
27,450,932.109
92,14,933.575
66,257,933.778
89,153,936.215
361,119,941.924
265,440,944.28
484,383,948.301
341,427,951.601
339,215,954.851
147,92,955.155
152,70,956.182
160,66,957.347
282,70,957.772
137,258,957.338
116,347,955.21
142,11,962.536
402,317,963.784
83,264,966.608
192,258,970.881
193,83,971.136
319,53,970.194
480,379,973.119
161,369,974.014
418,324,974.92
465,24,976.608
382,78,980.127
85,61,986.664
274,425,986.143
65,354,986.722
117,379,989.491
366,186,989.096
211,219,991.013
401,164,992.383
86,391,992.668
226,316,991.844
193,444,994.246
147,83,992.696
