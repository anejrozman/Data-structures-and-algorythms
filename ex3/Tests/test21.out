303,1366983
9,1042
7,969
6,562
5,267
5,210
5,67
4,221
4,182
4,179
3,147
3,129
3,54
2,40
2,36
2,36
2,36
2,32
2,16
2,14
2,9
2,6
2,6
2,3
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
